notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb

```diff
@@ -122,6 +122,8 @@
     "from azureml.core.image import ContainerImage\n",
```

Comment:
<p>suggest using slug style variables instead of camelcase</p><p><br></p><p>also, is rps standard? I've seen qps more often, but wasn't sure if that was industry specific</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/735/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
sure, i'll update to slug. 

Don't know if rps is standard, I just re-used what was in the [docs page](https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-deploy-and-where#autoscaling). Let's use qps if you think it's more common

Reply:
<p>fixed.</p>